src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,37 +1,119 @@
 import { useState } from 'react';
+
 import { TextField } from '../TextField';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  onAdd: (movie: Movie) => void;
+};
+
+type State = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+  const [state, setState] = useState<State>({
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  });
```

Comment:
```suggestion
  const [newMovie, setNewMovie] = useState<State>(DEFAULT_MOVIE);
```
- Create a default movie object so you can pass it here and then when resetting values
- `state` isn't informative name, you can have a lot of them